### PR TITLE
Fix diagonal degree matrix creation

### LINKED ISSUE #13: Question: GCN pytorch implementation
Thanks a lot for the code. Found it via your blog post about GCNs.

I see Kipf used Laplace symmetric normalisation - as you wrote also here: https://dsgiitr.com/blogs/gcn/ 

But I don't understand, why the diagonal degree matrix is produced using A instead of A_hat:
```
        self.A_hat = A+torch.eye(A.size(0))
        self.D     = torch.diag(torch.sum(A,1))  #<< should this not be self.A_hat?
        self.D     = self.D.inverse().sqrt()
```

### PR DESCRIPTION
**Pull request by byteSamurai**
This seems to be closes to Laplace symmetric normalisation according how Kipf and Welling it used, though it seems to have no effect in this example.

closes #13

### CODE REVIEW

**Review by AnirudhDagar — APPROVED** [review]
Thanks! 😄
